samples/.net/endpoints/batch/Contracts/BatchScoringProperties.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="BatchScoringProperties.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace BatchInferencingSamples
+{
+    internal class BatchScoringProperties
+    {
+        /// <summary>
+        /// Settings of the source data.
```

Comment:
Could we use standard comment style "Gets or sets..." - this clearly tells people teh nature of the property (gettable, settable) by seeing the intellisense.